### ISSUE TITLE
Create 5 or so different images - for the home page - to use of a reduced width montage

### ISSUE THREAD

**Issue by rrocchio** [universityofcalifornia/NeXt#140]
We are going to want the photo montage to reduce down in size - by those blocks using Media Query... Lauren can you create about 5 different sizes of the images - so that Keith can make the home page use the smaller images as the size of the window reduces down....

** I have requested UCSF to send me images from the last UCCSC conference - but if you create the images now, we can swap out the the files for Keith later...

**Comment by laurencullen** [comment]
Here is the photomontage at the 1024 size. More sizes to come. Please let me know any feedback.

![ucnext_images1024](https://cloud.githubusercontent.com/assets/13141571/9125952/622d1af0-3c5b-11e5-8372-6e12291c8571.jpg)

**Comment by laurencullen** [comment]
Here are more sizes. Please let me know if you need any more sizes or adjustments. (The sizes on titles are for the size of the image, not the breakpoint - the image isn't designed to be fully across the page.)

![ucnext_images384](https://cloud.githubusercontent.com/assets/13141571/9145377/a422b4ee-3d08-11e5-888c-18d371a05721.jpg)
![ucnext_images512](https://cloud.githubusercontent.com/assets/13141571/9145376/a4229770-3d08-11e5-8553-46c8ded10326.jpg)
![ucnext_images640](https://cloud.githubusercontent.com/assets/13141571/9145378/a4258944-3d08-11e5-88e7-2284332a4381.jpg)
![ucnext_images768](https://cloud.githubusercontent.com/assets/13141571/9145379/a428ed28-3d08-11e5-9057-bc3f2f82f50f.jpg)
![ucnext_images896](https://cloud.githubusercontent.com/assets/13141571/9145380/a42f74cc-3d08-11e5-8e2a-09cba686f43b.jpg)

**Comment by laurencullen** [comment]
sizes: all 128px high
width: 384, 512, 640, 768, 896, 1024

**Comment by krozett** [comment]
Thanks @laurencullen . Although you don't have to worry about resizing them, since it's essentially just a crop and I can do that with CSS.

When we have the new images, individual images are best, rather than a strip of them together. It doesn't matter that much right now, since they are just placeholders. But for the future...

**Comment by rrocchio** [comment]
@lcullen - Lauren- These are great!  Keith - interesting that you can just crop with CSS - I did not know you could do that!

**Comment by laurencullen** [comment]
@krozett Yes, I can provide individual images - please let me know when you need them. Also, please note that I moved some images around in the small size, so it's not just a crop from the center...

**Comment by rrocchio** [comment]
@ebollens  - can you weigh in on the best way to implement this photo montage - should we use the different width images - or should we use one and resize or "crop" it?
